src/presentation/http/router/note.ts

```diff
@@ -37,6 +37,11 @@ interface NoteRouterOptions {
    * Note service instance
    */
   noteService: NoteService,
+
+  /**
+   * Auth middleware
+   */
+  authMiddleware: preHandlerHookHandler;
```

Comment:
what if there will be many different middlewares in a future? maybe it would be better to pass some universal array, or import middlewares directly ?

Reply:
Change it by passing Middlewares object